src/Common/DigipolisPropertiesAwareInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Why is this interface here? A Trait (for which this is meant, looking at the methods) can't implement an interface.